vsintegration/tests/FSharp.Editor.Tests/Refactors/AddExplicitReturnTypeTests.fs

```diff
@@ -0,0 +1,176 @@
+﻿module FSharp.Editor.Tests.Refactors.AddExplicitReturnTypeTests
+
+open Microsoft.VisualStudio.FSharp.Editor
+open Xunit
+open NUnit.Framework
```

Comment:
Shouldn't be here, right?

Reply:
Thanks, yeah , unused usings and stuff doesnt seem to work for fsharp in vs2022